### PR TITLE
Keyboard layout

### PR DESCRIPTION
**Pull request by jpnurmi**
| Before | After |
|---|---|
| image | image |

Ref: #1618

### CODE REVIEW

File: packages/ubuntu_desktop_installer/test/keyboard_layout/keyboard_layout_page_test.dart [hunk 1]
```diff
@@ -85,12 +85,14 @@ void main() {
...
-      await tester.tap(variantTile);
+      await tester.tap(find.byType(MenuButtonBuilder<int>));
+      await tester.pumpAndSettle();
+      final variantItem = find.widgetWithText(MenuItemButton, 'Variant $i');
+      expect(variantItem, findsOneWidget);
+      await tester.tap(variantItem);
       verify(model.selectVariant(i)).called(1);
     }
-  });
+  }, skip: true); // TODO: fix the test
```
Comment (jpnurmi) [inline packages/ubuntu_desktop_installer/test/keyboard_layout/keyboard_layout_page_test.dart:95]: i'm having trouble figuring out why `tap(MenuButtonBuilder)` doesn't work :(

```

Warning: A call to tap() with finder "exactly one widget with type "MenuButtonBuilder<int>" (ignoring offstage widgets): MenuButtonBuilder<int>(dependencies: [Directionality, MediaQuery, _InheritedTheme, _LocalizationsScope-[GlobalKey#79b53]], state: _MenuButtonBuilderState<int>#71692)" derived an Offset (Offset(609.0, 451.0)) that would not hit test on the specified widget.
Maybe the widget is actually off-screen, or another widget is obscuring it, or the widget cannot receive pointer events.
```

Comment (d-loose) [reply packages/ubuntu_desktop_installer/test/keyboard_layout/keyboard_layout_page_test.dart:95]: it works when I only test the very first variant without a loop - I suspect the issue is that the test tries to tap the `MenuButtonBuilder<int>` again, while the popup is visible

Comment (d-loose) [reply packages/ubuntu_desktop_installer/test/keyboard_layout/keyboard_layout_page_test.dart:95]: Moving
```dart
await tester.tap(find.byType(MenuButtonBuilder<int>));
await tester.pumpAndSettle();
```
out of the loop works

Comment (jpnurmi) [reply packages/ubuntu_desktop_installer/test/keyboard_layout/keyboard_layout_page_test.dart:95]: ahh, i'm an idiot :D thank you!

**Review by d-loose — APPROVED** [review]
Ah, that's the better solution of course!

LGTM :+1:

**Comment by jpnurmi** [comment]
it's funny that in the test it's indeed possible to click through all items before the popup closes itself (next frame?)

**Pull request merged**